Check for NPC on ownedby condition

SplenectomY opened the pull request:
Added a check to Keen's mysterious NPC Identity list, since NPCs cannot be found with MySession.Static.Players.TryGetPlayerById

SplenectomY committed:
Check for NPC on ownedby condition

Added a check to Keen's mysterious NPC Identity list, since NPCs cannot be found with MySession.Static.Players.TryGetPlayerById

SplenectomY committed:
Update CleanupModule.cs

SplenectomY committed:
Update CleanupModule.cs

SplenectomY committed:
Update CleanupModule.cs

rexxar-tc commented on Essentials/Commands/CleanupModule.cs:
```diff
@@ -280,8 +280,19 @@ public bool OwnedBy(MyCubeGrid grid, string str)
+                    if(long.TryParse(str, out long NPCId))
+                    {
+                        if(MySession.Static.Players.IdentityIsNpc(NPCId))
+                        {
```
You'll get a null reference exception if this statement is false. Remove the else on L291

SplenectomY committed:
Complying to change req.

https://github.com/TorchAPI/Essentials/pull/55/files/970ab4ee61c9cc6eb2f4b082ff7ff00a5d71fda8
